src/SelectNext.tsx

```diff
@@ -160,7 +160,7 @@ function NonMemoizedNonForwardedSelect<T extends SelectProps.Option[]>(
                     },
                     ...options
                 ].map((option, index) => (
-                    <option {...option} key={`${option.value}-${index}`}>
+                    <option value={option.value} key={`${option.value}-${index}`}>
```

Comment:
Not sure about that. Even if it's not officially supported by the type system it's nice to enable any option to be provided don't you think?

Reply:
I checked dsfr documentation and they provide only value to html option. Do you know why you provided all option value ? I don't think it's useful.

Reply:
Je ne sais pas, peut y avoir des attributs genre `aria-*`... 

Mais surtout l'idée c'est de changer qu'une seul chose par PR.  Je pense que c'est très bien que le label soit un ReactNode  mais es-ce qu'on a besoin de changer `{...option}` pour ça?  

Anyway. Feel free to merge tu n'a pas a attendre mon feu vert, je donne juste mon avis.

Reply:
Le type d'`option` est : 
```ts
export type Option<T extends string = string> = {
        value: T;
        label: ReactNode;
        disabled?: boolean;
        /** Default: false, should be used only in uncontrolled mode */
        selected?: boolean;
    };
```
Comment tu peux avoir des attributs aria ? Je dois manquer quelque chose